esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -291,7 +291,7 @@ void loop()
         udp.endPacket();
     }
 #elif (PROTOCOL_TYPE == 2) // Bluetooth
-    uint8_t idx = 0;
+    uint16_t idx = 0;
```

Comment:
make it int, this seems to be a preferred practice in Arduino. Also, some functions return tsize_t which is int.